pygeoapi/models/provider/base.py

```diff
@@ -74,34 +74,692 @@ class TileMatrixSetEnumType(BaseModel):
     tileMatrixSet: str
     tileMatrixSetURI: str
     crs: str
-    tileMatrixSetDefinition: dict
-
+    title: str
+    orderedAxes: List[str]
+    wellKnownScaleSet: str
+    tileMatrices: List[dict]
 
 class TileMatrixSetEnum(Enum):
     WORLDCRS84QUAD = TileMatrixSetEnumType(
         tileMatrixSet="WorldCRS84Quad",
         tileMatrixSetURI="http://www.opengis.net/def/tilematrixset/OGC/1.0/WorldCRS84Quad",  # noqa
         crs="http://www.opengis.net/def/crs/OGC/1.3/CRS84",
-        tileMatrixSetDefinition=
-            {
-                'type': 'application/json',
-                'rel': 'http://www.opengis.net/def/rel/ogc/1.0/tiling-scheme',
-                'title': 'WorldCRS84QuadTileMatrixSet definition (as JSON)',
-                'href': 'https://raw.githubusercontent.com/opengeospatial/2D-Tile-Matrix-Set/master/registry/json/WorldCRS84Quad.json'  # authoritative TMS definition
-            }
-        )
+        title="World Mercator WGS84 (ellipsoid)",
```

Comment:
Is this entire definition available on schemas.opengis.net? Should we be baking artifacts (with values) in the code?  Is this better off in a resource file of some sort?

Reply:
@tomkralidis if the goal is to get rid of a `pydantic` definition I would be inclined to keep this since it is aligned with the rest of the tiles implementation and procrastinate the removal of the whole pydantic code once we have the consensus of the community.

Reply:
@tomkralidis the only normative definition is on the OGC definition server, but unfortunately [that is not available](https://github.com/opengeospatial/NamingAuthority/issues/198). It seems everyone else is hosting definitions on their server, as recommendation of the spec.

Reply:
https://test.cubewerx.com/cubewerx/cubeserv/demo/ogcapi/Daraa/tileMatrixSets
https://maps.gnosis.earth/ogcapi/tileMatrixSets
https://demo.ldproxy.net/daraa/tileMatrixSets/

Reply:
@tomkralidis routes for django were added here: https://github.com/geopython/pygeoapi/pull/1497/commits/5931e1a432d7a2843d8d4584cac5447ab98349a3